cpp/example_code/transcribe/README.md

```diff
@@ -0,0 +1,38 @@
+# Amazon Transcribe code examples for the C++ SDK
+## Overview
+The code examples in this directory demonstrate how to work with the Amazon Transcribe using the AWS SDK for C++.
+
+Amazon Transcribe is an automatic speech recognition service that makes it easy to add speech to text capabilities to any application.
+## ⚠️ Important
+* Running this code might result in charges to your AWS account.
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+### Scenarios
+
+- [Transcribing from an audio file](./get_transcript.cpp) - Shows streaming transcription by mimicking live audio using a flat file. This example uses the following API classes:
+   
+   - TranscribeStreamingServiceClient
+   - StartStreamTranscriptionHandler
+   - StartStreamTranscriptionRequest
+
+## Running the Examples
+Before using the code examples, first complete the installation and setup steps of [Getting Started](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started.html) in the AWS SDK for C++ Developer Guide.
+The Getting Started section covers how to obtain and build the SDK, and how to build your own code utilizing the SDK with a sample "Hello World"-style application. 
```

Comment:
The Getting Started section --> This section
Remove quotation marks from Hello World.

Reply:
changed